scm-core/src/main/java/sonia/scm/repository/api/ModificationsCommandBuilder.java

```diff
@@ -72,15 +72,34 @@
   @Setter
   private boolean disablePreProcessors = false;
 
+  /**
+   * Set this to compute either the midifications of the given revision, or additionally set
+   * {@link #baseRevision(String)} to compute the modifications between this and the
+   * other revision.
+   * @return This command builder.
+   */
```

Comment:
Missing since tag

Reply:
This is the old method. I only added a JavaDoc.